rustsec/src/advisory.rs

```diff
@@ -88,6 +88,15 @@ impl Advisory {
     pub fn withdrawn(&self) -> bool {
         self.metadata.withdrawn.is_some()
     }
+
+    /// License under which this advisory is available, represented as SPDX identifier.
+    pub fn license(&self) -> Option<&str> {
+        match self.metadata.syndicated_from {
+            Some(Origin::GithubAdvisoryDatabase) => Some("CC-BY-4.0"),
+            Some(Origin::Other(_)) => None,
```

Comment:
Not sure what the right interface is in case of unknown origin. Maybe using a `Result` instead?

Reply:
Personally I think `None` is OK here. The problem is that the license isn't known, not that it's necessarily invalid, and I think `Option` reflects that.

Reply:
Or maybe even not provide it, and only render license on the website.

Reply:
There is no multi-license databases except us after this. Everyone else is CC-BY-4.0.

I agree - let's build features when we need them.